s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationTest.java

```diff
@@ -248,8 +250,13 @@ void avroTest(final TestInfo testInfo) throws IOException {
                 .contains(entry(1, "Hello, Kafka Connect S3 Source! object 1"),
                         entry(2, "Hello, Kafka Connect S3 Source! object 2"));
 
-        // Verify offset positions
         verifyOffsetPositions(offsetKeys, 5);
```

Comment:
Please remove this line to make the tests passing.

Reply:
forgot to remove it, but its still the same.